designer/server/src/main/scala/pl/touk/nussknacker/ui/process/periodic/PeriodicProcessService.scala

```diff
@@ -62,44 +63,6 @@ class PeriodicProcessService(
 
   private implicit val localDateTimeOrdering: Ordering[LocalDateTime] = Ordering.by(identity[ChronoLocalDateTime[_]])
 
-  def getScenarioActivitiesSpecificToPeriodicProcess(
```

Comment:
Instead of returning `on demand` the `PerformedScheduledExecution` activities for a given scenarion, based on `scheduled_deployments` db table:
- this `PeriodicProcessService` actually saves activities to the `scenario_activities` db table
- old deployments are migrated and saved as activities (db migration 68)